docker_build.sh

```diff
@@ -0,0 +1,2 @@
+docker build --platform linux/amd64 -t us-docker.pkg.dev/fw-ai-cp-prod/inference/fireworks-text-completion-poe-bot:latest -f ./Dockerfile .
+docker push us-docker.pkg.dev/fw-ai-cp-prod/inference/fireworks-text-completion-poe-bot:latest
```

Comment:
Remember that this repo is public so don't refer to our internal repositories. Just make this build a tag locally and we can tag and push manually